app/src/main/java/com/hihi/twiliosms/EncodeBase64.java

```diff
@@ -0,0 +1,11 @@
+package com.hihi.twiliosms;
+
+import android.util.Base64;
+
+class EncodeBase64 {
+
+    static String encodeCredentials(String sid, String token) {
+        return String.format("Basic %s", Base64.encodeToString((sid + ":" + token).getBytes(), Base64.NO_WRAP));
```

Comment:
Static imports and `String#format`:
`return String.format("Basic %s", Base64.encodeToString((String.format("%s:%s", sid, token)).getBytes(), Base64.NO_WRAP));`